core/system/inc/linker.h

```diff
@@ -121,6 +121,9 @@ typedef struct {
     UvisorLibHooks const * const lib_hooks;
 
     TUvisorDebugDriver const * const debug_driver;
+
+    /* Stack limit for MSPNS_LIM */
```

Comment:
Change MSPNS_LIM to MSPLIM_NS in the comment.

Reply:
fix